litellm/proxy/proxy_server.py

```diff
@@ -3496,6 +3496,13 @@ async def chat_completion(
         if "api-version" in query_params:
             data["api_version"] = query_params["api-version"]
 
+        # Allow OpenAI and Anthropic's auth headers to be used as api_key
+        if litellm.use_llm_key_in_header or 'api_key' not in data:
+            if auth_bearer := request.headers.get('AUTHORIZATION'):  # OpenAI
+                data['api_key'] = _get_bearer_token(auth_bearer)
+            elif value_of_x_api_key := request.headers.get('X-API-KEY'):  # Anthropic
```

Comment:
this endpoint is in the openai spec - can you remove this snippet. 

We expect the auth header to be in the bearer token

Reply:
Thank you for the feedback, removed the `'X-API-KEY'` section.

> Also in few line pycharm removed trailing space, i kept that in the commit.